### PR TITLE
MultiServer: Improve command response to help troubleshooting

### PR DESCRIPTION
**Pull request by chandler05**
## What is this fixing or adding?
Adjusts the response of commands when a match is not found for an argument by making it more clear what was interpreted as the argument.

## How was this tested?
Quick testing by me on the WebHost.

## If this makes graphical changes, please attach screenshots.
Multiworld 73ZMrsBFTWW9kTbwRQtcog - Google Chrome 2_16_2024 12_02_22 PM

### CODE REVIEW

File: MultiServer.py [hunk 1]
```diff
@@ -1050,17 +1050,18 @@ def get_intended_text(input_text: str, possible_answers) -> typing.Tuple[str, bo
...
         else:
-            return picks[0][0], False, f"Too many close matches, did you mean {picks[0][0]}? ({picks[0][1]}% sure)"
+            return picks[0][0], False, f"Too many close matches to '{input_text}', " \
+                                       f"did you mean '{picks[0][0]}'? ({picks[0][1]}% sure)"
     else:
         if picks[0][1] > 90:
             return picks[0][0], True, "Only Option Match"
         else:
-            return picks[0][0], False, f"Did you mean {picks[0][0]}? ({picks[0][1]}% sure)"
+            return picks[0][0], False, f"Did you mean '{picks[0][0]}'? ({picks[0][1]}% sure)"
```
Comment (agilbert1412) [inline MultiServer.py:1064]: Shouldn't this one mention the input text as well?

**Review by agilbert1412 — CHANGES_REQUESTED** [review]

File: MultiServer.py (hunk 1 above)
Comment (chandler05) [reply MultiServer.py]: :+1:

File: MultiServer.py [hunk 2]
```diff
@@ -1050,17 +1050,19 @@ def get_intended_text(input_text: str, possible_answers) -> typing.Tuple[str, bo
...
         elif picks[0][1] < 75:
-            return picks[0][0], False, f"Didn't find something that closely matches, " \
-                                       f"did you mean {picks[0][0]}? ({picks[0][1]}% sure)"
+            return picks[0][0], False, f"Didn't find something that closely matches '{input_text}', " \
+                                       f"did you mean '{picks[0][0]}'? ({picks[0][1]}% sure)"
         elif dif > 5:
             return picks[0][0], True, "Close Match"
         else:
-            return picks[0][0], False, f"Too many close matches, did you mean {picks[0][0]}? ({picks[0][1]}% sure)"
+            return picks[0][0], False, f"Too many close matches to '{input_text}', " \
```
Comment (black-sliver) [inline MultiServer.py:1058]: Not a native speaker, but "to" sounds wrong to me here..
```suggestion
            return picks[0][0], False, f"Too many close matches for '{input_text}', " \
```

Comment (chandler05) [reply MultiServer.py:1058]: I think both are correct, but "to" is usually used in the context of something being similar to something, like it is in this case.

Comment (agilbert1412) [reply MultiServer.py:1058]: I think I prefer "for", personally, but I do believe both are valid and "good enough"

Comment (chandler05) [reply MultiServer.py]: Alright, sounds good to me. :+1:

**Review by black-sliver — APPROVED** [review]
Thanks!

**Pull request merged**